src/documents/CoreAPI.md

```diff
@@ -345,6 +367,29 @@ var CATTyears = fluid.hashToArray(hash, "species", function (newElement, oldElem
 Converts an array consisting of a mixture of arrays and non-arrays into the concatenation of any inner arrays 
 with the non-array elements. The original array *** will not be modified ***. See description of [mapcat](http://martinfowler.com/articles/collection-pipeline/flat-map.html) or `flat-map`.
 
+### fluid.freezeRecursive(tofreeze)
+
+* `tofreeze {Array|Object}` The object to be frozen recursively
+* Returns: `{Array|Object}` The supplied argument, recursively frozen
+
+Calls [`Object.freeze`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze) at each level of containment of the supplied object.
+This function is a no-op if a primitive value is supplied.
+
+### fluid.pushArray(holder, member, topush)
```

Comment:
I've added a sentence to clarify that the member must either be undefined or hold an array.